### PR TITLE
Stop using Travis CI, start using GitHub Actions CI

### PR DESCRIPTION
**Pull request by etagwerker**
This PR updates CI configuration for this gem.

### CODE REVIEW

**Comment by axlekb** [comment]
Any chance we can get this PR and the CI back up and running? `launchy 3.0.0` is out, and this would be great to have this change in before relaxing the gemspec version constraint:

```
  s.add_dependency "launchy", "~> 2.1"
  ```

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+jobs:
+  test-ruby:
+    runs-on: ${{ matrix.os }}-latest
+    strategy:
+      matrix:
+        os: [ubuntu]
+        ruby-version: ["2.6", "2.7", "3.0", "3.1", "3.2"]
+    steps:
+      - uses: actions/checkout@v2
```
Comment (mjankowski) [inline .github/workflows/main.yml:20]: You may want to bump this to `v4` - https://github.com/actions/checkout/releases/tag/v4.0.0

Will avoid node version deprecation warning on CI runs.

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    branches:
+    - main
+
+jobs:
+  test-ruby:
+    runs-on: ${{ matrix.os }}-latest
+    strategy:
+      matrix:
+        os: [ubuntu]
+        ruby-version: ["2.6", "2.7", "3.0", "3.1", "3.2"]
```
Comment (mjankowski) [inline .github/workflows/main.yml:18]: There's no harm in supporting these older versions (looks like the gemspec doesn't state a hard lowest ruby version dep?), but just wanted to note:

- Ruby 3.0 and earlier are EOL
- Maybe add 3.3 to this matrix?

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Ruby ${{ matrix.ruby-version }}
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby-version }}
+          bundler-cache: true
+      - name: Build and run tests
+        run: |
+          gem install bundler
```
Comment (mjankowski) [inline .github/workflows/main.yml:28]: I'm 99% sure that just having the above `setup-ruby` action in place with `bundler-cache: true` is going to have already run `bundle install` for you ... this whole step run section could be just the rspec run.

**Pull request merged**